modules/core/quick/farm_quick.links.menu.yml

```diff
@@ -1,3 +1,8 @@
 farm.quick:
   class: Drupal\Core\Menu\MenuLinkDefault
   deriver: Drupal\farm_quick\Plugin\Derivative\QuickFormMenuLink
+farm.quick_setup:
+  title: Quick Forms
+  description: Quick forms make it easy to record common activities.
+  parent: farm.setup
```

Comment:
We need to add a dependency on `farm_setup` with this commit.

Reply:
I pushed a fixup commit for this: https://github.com/farmOS/farmOS/pull/785/commits/205b88502027621364abcfd85bc3368c72d14a10

Reply:
Squashed